packages/ui/react-ui/__tests__/CrossmintPayButton.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { render, fireEvent, screen, act } from "@testing-library/react";
+import "@testing-library/jest-dom";
+
+import { CrossmintPayButton } from "../src/CrossmintPayButton";
+import { LIB_VERSION } from "../src/version";
+
+// Todo: create a global service for this to work everywhere and to be able to customize resolved/rejected responses
```

Comment:
1. Can you name this as "TODO" ?

We are trying to standardize the way we write it so it's easy to set automations and rules around it

2. Can you file an issue on GH and attach the issue number here?

Example:

`// TODO(#777): create a global service for this to work everywhere and to be able to customize resolved/rejected responses`